src/gen-ts.ts

```diff
@@ -552,6 +556,27 @@ function handleProperties(analyzerProperties: Iterable<analyzer.Property>):
   return tsProperties;
 }
 
+function generateObjectMap(feature: analyzer.Element): ts.Property[] {
```

Comment:
Add short description, e.g. "Create a signature for the`$` node ID map on a Polymer element."